examples/knative/powershell/kn-ps-vrni-databus/README.md

```diff
@@ -0,0 +1,177 @@
+
+```console
+export TAG=<version>
+docker build -t <docker-username>/kn-ps-vrni-databus:${TAG} .
+```
+
+# Step 2 - Test
+
+Verify the container image works by executing it locally.
+
+Change into the `test` directory
+```console
+cd test
+```
+
+Update the following variable names within the `docker-test-env-variable` file and either the `send-local-webhook-test.ps1` (PowerShell Script) or `send-local-webhook-test.sh` (Bash Script)
+
+* WEBHOOK_SINK_URL - The sink URL (e.g. http://default-broker-ingress.vmware-functions.svc.cluster.local) to send the CloudEvent (leave blank for default internal URL)
+
+Start the container image by running the following command:
+
+```console
+docker run -e SERVICE_TEST=true -e SERVICE_DEBUG=true -e PORT=8080 --env-file docker-test-env-variable -it --rm -p 8080:8080 <docker-username>/kn-ps-vrni-databus:${TAG}
+```
+
+In a separate terminal, run either `send-local-webhook-test.ps1` (PowerShell Script) or `send-local-webhook-test.sh` (Bash Script) to simulate a payload being sent to the local container image
+
+```console
+Testing Function ...
+See docker container console for output
+
+# Output from docker container console
+Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
+--     ----            -------------   -----         -----------     --------             -------
+2      Job2            ThreadJob       NotStarted    False           PowerShell           …
+11/04/2021 11:19:01 - PowerShell HTTP server start listening on 'http://*:8080/'
+11/04/2021 11:19:01 - Processing Init
+
+11/04/2021 11:19:01 - Init Processing Completed
+
+11/04/2021 11:19:03 - Unable to get remote client, setting CloudEvent source to function hostname
+11/04/2021 11:19:03 - [Headers]:
+
+Name                           Value
+----                           -----
+Accept                         {*/*}
+User-Agent                     {curl/7.77.0}
+Content-Length                 {3750}
+Host                           {localhost:8080}
+Content-Type                   {application/json}
+
+
+11/04/2021 11:19:03 - [Body]:
+
+{	"EntityMessageList": [..big array of data..]}
+
+11/04/2021 11:19:03 - Creating CloudEvents...
+
+11/04/2021 11:19:03 - Running in local development: not sending CloudEvent to VMware Event Broker
+```
+
+# Step 3 - Deploy
+
+> **Note:** The following steps assume a working Knative environment using the
+`default` Rabbit `broker`. The Knative `service` and `trigger` will be installed in the
+`vmware-functions` Kubernetes namespace, assuming that the `broker` is also available there.
+
+Push your container image to an accessible registry such as Docker once you're done developing and testing your function logic.
+
+```console
+docker push <docker-username>/kn-ps-vrni-databus:${TAG}
+```
+
+---
+> **Note:** This step is only required when using multiple broker appliances and you want to send the Cloud Events to another broker.
```

Comment:
nit: can you please drop this comment and reword the following paragraph so it's easier to understand?

E.g. "If you deploy this function to a VEBA environment, not change is needed to the webhook_config.json. In all other cases, update the file with the sink address where the function should send the CloudEvent to."